metadata.rb

```diff
@@ -12,12 +12,14 @@
 supports 'amazon'
 supports 'fedora'
 supports 'debian', '>= 7.0'
-supports 'ubuntu', '>= 12.04'
+supports 'ubuntu', '>= 14.04'
 
 supports 'opensuse', '>= 13.0'
 supports 'opensuseleap'
 supports 'suse', '>= 12.0'
 
+depends 'selinux'
```

Comment:
if this is for test then move it over to the test cookbook. No reason to have people pull it down otherwise

Reply:
good point, will fix.  and still a lot of weird systemd failures in tests I need to figure out